checkout.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+    <section id="banner">
+      <div id="letterhead">
+        <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+      <div id="cart">
+        <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo"><p id="cartprice">$1234.56</p>
+        <div id="checkout-button">
+          <p>Checkout</p>
+        </div>
+      </div>
+    </section>
+    <section id="subbanner">
+      <div >
+        <p><a class="subtitle" href="index.html">Home&nbsp;&nbsp;/</a><a class="subtitle" href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a><a class="subtitle" href="blog.html">&nbsp;&nbsp;Blog</a></p>
+      </div>
+    </section>
+    <section id="welcome">
+      <div>
+        <h2>Checkout</h2>
+      </div>
+    </section>
+    <section>
+        <div class="formfield">
```

Comment:
In your formfield I would add a couple items.  I would style the labels to have similar coloring as the theme.  In addition, I might change their fonts.

Reply:
Another thing you might want try is making a type of container to hold city, state, and zip in the same line as you style this page.